tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -563,3 +563,22 @@ Feature: Sharing files and folders with internal users
     And the user opens the share dialog for folder "simple-folder" using the webUI
     Then user "User Two" should be listed without additional info in the collaborators list on the webUI
 
+  Scenario: collaborators list contains the current user when he is an owner
```

Comment:
always use the singular they: https://en.wikipedia.org/wiki/Singular_they

Reply:
Adjusted all the scenarios titles. Pls re-check if I got it right :wink: